### DISCUSSION TITLE
tr

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#13388] [Docs]
# tr

Discover Ultralytics YOLOv8 - the latest in real-time object detection and image segmentation. Learn its features and maximize its potential in your projects.

https://docs.ultralytics.com/tr

**Comment by furkanbasersk8** [comment]
Perfect Article Thanks for Ali and Joseph Redmon

**Reply by pderrenger** [reply]
Thank you for your kind words! Joseph Redmon and Ali Farhadi indeed made significant contributions to the field with the development of YOLO. We're glad you found the article informative and helpful. If you have any more questions or need further assistance, feel free to ask. Happy detecting! 🚀
